docs/deploying/k8s.md

```diff
@@ -0,0 +1,98 @@
+# Setting up and using Kubernetes
+
+## Install `kubectl`
+
+There are [many ways to install kubctl](https://kubernetes.io/docs/tasks/tools/install-kubectl/). You may also wish to refer to the commands in the formula too:
+https://github.com/elifesciences/builder-base-formula/blob/master/elife/kubectl.sls
+
+On Ubuntu I used snap:
+
+```
+sudo snap install kubectl --classic
+```
+
+Test the client installation of `kubectl`:
+
+```
+kubectl version
+```
+
+This should report the client version. But don't worry if there are errors for the server version as this is not configured yet.
+
+### Install aws-iam-authenticator
+
+Change the cwd to the place you wish to install and follow the following:
+
+```
+cd ~/.local/bin
+wget https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-07-26/bin/linux/amd64/aws-iam-authenticator
+chmod +x aws-iam-authenticator
+aws-iam-authenticator
+```
+
+### Make sure your `awscli` is the correct version
+
+The version of `awscli` should be >= 1.16
+
+Check by running, and update if needed:
+
+```
+aws --version
+```
+
+### Update your local config to use the AWS cluster
+
+```
+aws eks update-kubeconfig --name kubernetes--demo --role-arn arn:aws:iam::512686554592:role/kubernetes--demo--AmazonEKSUserRole
```

Comment:
This will change, but for a IAM user to be able to assume this role, it needs to be whitelisted at https://github.com/elifesciences/terraform-eks/blob/master/user-roles.tf